pkg/trace/info/info.go

```diff
@@ -196,10 +196,22 @@ type infoString string
 
 func (s infoString) String() string { return string(s) }
 
+type infoVersion struct {
```

Comment:
nit: this struct is only used in `InitInfo`, so (even though unusual) you could declare it inside the function body (it is possible to declare a type inside a function body). I've also seen this pattern when a struct is a "use and dump":
```go
return struct {
	Version   string
	GitCommit string
}{
	Version:   conf.AgentVersion,
	GitCommit: conf.GitCommit,
}
```